bldsys/cmake/component_helpers.cmake

```diff
@@ -59,6 +73,5 @@ function(vkb__register_component)
         endif()
```

Comment:
Adding
`        set_property(TARGET vkb__${TARGET_NAME} PROPERTY FOLDER "components")`
would list this interface library in the solution